test/index.ts

```diff
@@ -189,6 +192,154 @@ describe('Storage', () => {
     });
   });
 
+  describe('hmacKey', () => {
+    it('should throw if accessId is not provided', () => {
+      assert.throws(() => {
+        storage.hmacKey();
+      }, /access ID is needed/);
```

Comment:
Could you check the full error message?: `An access ID is needed to create an HmacKey object.`